lib/routes/token.js

```diff
@@ -16,9 +17,20 @@ const P = require('../promise');
 const Scope = require('../scope');
 const validators = require('../validators');
 
+const HEX_STRING = validators.HEX_STRING;
+
 const MAX_TTL_S = config.get('expiration.accessToken') / 1000;
 const GRANT_AUTHORIZATION_CODE = 'authorization_code';
 const GRANT_REFRESH_TOKEN = 'refresh_token';
+const GRANT_JWT = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
+
+const JWT_AUD = config.get('publicUrl') + '/v1/token';
```

Comment:
Is it usual to have audiences with a path component?  I don't recall seeing this pattern used elsewhere.  I guess it can't hurt though...